cli/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "piston-cli",
-    "version": "1.0.0",
```

Comment:
any reason for a version bump?

Reply:
It is the package-lock.json file. It was updated automatically.